### DISCUSSION TITLE
Splash Image? Among others...

### DISCUSSION THREAD

**Discussion by Durahl** [EdgeTX/edgetx#1924] [Q&A] [answered]
Greeting!

I _just_ finished my **OpenTX** to **EdgeTX** ( _and_ [FrSky Horus X12S Touch Screen](https://wimalopaan.github.io/Electronics/rc/touch.html) ) conversion and am now in the progress of configuring it to my liking with some questions having popped up since:
- Can the EdgeTX **Splash Screen** still be changed? I tried doing it [like in this video](https://youtu.be/EEmsMJbhCfE?t=437) but it won't use _mine_ 🤔
- How _exactly_ does the Color Mixer in **Widget Settings** work? Is this HSB instead of RGB like in the Themes Editor? 🤨
I just can't seem to get a grey color to show up when working with those numbers 😑

Thanks in Advance! 😁

**Comment by Durahl** [comment, marked as answer]
Never mind... I didn't realize all images need to be ***.PNG** 😑

**Reply by pfeerick** [reply]
Colour picker in widget options is a bit odd, as colorlcd displays work in RGB565 mode, and that was the only colour picker initially available on radio. This may help with picking a colour... you'll see that you can double the green to bring it into alignment with red and blue ... https://chrishewett.com/blog/true-rgb565-colour-picker/

**Reply by Durahl** [reply]
Oooh... Now it's coming together... The center value ( green ) having twice the resolution kinda threw me off 🤔
Thanks for explaining!

**Comment by newriderca** [comment]
Greeting!

I _just_ finished my **OpenTX** to **EdgeTX** ( _and_ [FrSky Horus X12S Touch Screen](https://wimalopaan.github.io/Electronics/rc/touch.html) ) conversion and am now in the progress of configuring it to my liking with some questions having popped up since:

* Can the EdgeTX **Splash Screen** still be changed? I tried doing it [like in this video](https://youtu.be/EEmsMJbhCfE?t=437) but it won't use _mine_ :thinking:

* How _exactly_ does the Color Mixer in **Widget Settings** work? Is this HSB instead of RGB like in the Themes Editor? raised_eyebrow
      I just can't seem to get a grey color to show up when working with those numbers :expressionless:

 
Thanks in Advance! :grin:

![310917343_10229671774559540_1895599872356786521_n](https://user-images.githubusercontent.com/115674113/195472362-32a41a9f-6824-44f8-be89-b276ec111ad9.jpg)
 
I have Gentoo ;) as boot pic. To bad I cant modified of how long it can run or attach a sound to it with same length or use gif.

**Reply by pfeerick** [reply]
> Can the EdgeTX Splash Screen still be changed? I tried doing it [like in this video](https://youtu.be/EEmsMJbhCfE?t=437) but it won't use mine 🤔

Yes, you just need to have the image in the correct place  - `/IMAGES/splash.png`

> How exactly does the Color Mixer in Widget Settings work? Is this HSB instead of RGB like in the Themes Editor? raised_eyebrow
I just can't seem to get a grey color to show up when working with those numbers 😑

Well, there are two buttons there - one says RGB, the other says HSV ... So you have the choice of both ;) Unless the firmware you're using is still an old build of 2.8, as the color picker was improved quite a bit about 6 weeks ago. 

> To bad I cant modified of how long it can run or attach a sound to it with same length 

Well, it's a good thing you suggested it as a possible enhancement, then isn't it ;)

**Comment by gagarinlg** [comment]
You just have to exchange the splash.png
